eng/Versions.props

```diff
@@ -167,7 +167,7 @@
     <optimizationPGOCoreCLRVersion>1.0.0-prerelease.22511.6</optimizationPGOCoreCLRVersion>
     <!-- Not auto-updated. -->
     <MicrosoftDiaSymReaderNativeVersion>16.9.0-beta1.21055.5</MicrosoftDiaSymReaderNativeVersion>
-    <SystemCommandLineVersion>2.0.0-beta4.22355.1</SystemCommandLineVersion>
+    <SystemCommandLineVersion>2.0.0-beta4.22564.1</SystemCommandLineVersion>
```

Comment:
System.CommandLine is also listed in https://github.com/dotnet/runtime/blob/main/eng/Version.Details.xml#L83-L86 . Do we need to update it there as well?

Also, it does not sound right that this is in the "Not auto-updated." block. I would think that everything in Version.Details.xml should is auto-updated.

Reply:
We don't have a subscription set up to auto-update System.CommandLine, and as a result the entry in Version.Details.xml doesn't actually do anything. We can add a subscription though and make this auto-update.

Reply:
> We can add a subscription though and make this auto-update.

@jkoritzinsky Could you please add it?

I see that dotnet/sdk has a System.CommandLine subscription (https://github.com/dotnet/sdk/pull/29131). We need to be on a plan where all repos contributing to a product have a subscription for a thing, or none have a subscription. Otherwise, we will end up with build breaks in source-build leg when the bits meet in the installer.

Reply:
After this PR is merged, I'll create and enable a subscription for System.CommandLine.

Reply:
> subscription for System.CommandLine.

Btw, it's pulled from `dotnet-libraries` feed in NuGet.config at root.

Reply:
> Could you please add it?

We are definitely going to do that, but when the time comes. Currently we are merging too many small breaking changes into S.CL. We would need to update SDK/runtime once a day to account for that.

My current plan is to gather a big batch of breaking changes (there will be a blocked PR in the SDK repo with dependencies update), get the first part of S.CL API approved by the API Review Board and then merge the changes into SDK and runtime.

Reply:
At one time S.CL was also flowing into Templating.  Please make sure any other subscriptions are also disabled.